Prog/Subsidized_unit_counts.sas

```diff
@@ -0,0 +1,594 @@
+    n='Projects'    
+    sum='Assisted Units' * LIHTC_all_assistedunits=' ' 
+    ;
+  format LIHTC_endyr yearrng.;
+run;
+
+ods csvall close;
+
+/*LIHTC-15year*/
+ods csvall body="&_dcdata_default_path\RegHsg\Prog\Subsidized_unit_counts_LIHTC15yr.csv";
+
+title3 "LIHTC projects and assisted units with subsidies expiring 2015 - 2035, 15 year dates";
+footnote1;
+
+proc tabulate data=Work.ConstructionDates format=comma10. noseps missing;
+  where not missing(LIHTC_15yr);
+  class LIHTC_15yr;
+  var LIHTC_all_assistedunits;
+  table 
+    /** Rows **/
+    all='Total' LIHTC_15yr=' ',
+    /** Columns **/
+    n='Projects'    
+    sum='Assisted Units' * LIHTC_all_assistedunits=' ' 
+    ;
+  format LIHTC_15yr yearrng.;
+run;
+
+ods csvall close;
+
+/*RHS515*/
+ods csvall body="&_dcdata_default_path\RegHsg\Prog\Subsidized_unit_counts_rhs515.csv";
+
+title3 "RHS 515 projects and assisted units with subsidies expiring 2015 - 2035";
+footnote1;
+
+proc tabulate data=Work.ConstructionDates format=comma10. noseps missing;
+  where not missing(RHS515_endyr);
+  class RHS515_endyr;
+  var RHS515_all_assistedunits;
+  table 
+    /** Rows **/
+    all='Total' RHS515_endyr=' ',
+    /** Columns **/
+    n='Projects'    
+    sum='Assisted Units' * RHS515_all_assistedunits=' ' 
+    ;
+  format RHS515_endyr yearrng.;
+run;
+
+ods csvall close;
+
+/*RHS538*/
+ods csvall body="&_dcdata_default_path\RegHsg\Prog\Subsidized_unit_counts_rhs538.csv";
+
+title3 "RHS 538 projects and assisted units with subsidies expiring 2015 - 2035";
+footnote1;
+
+proc tabulate data=Work.ConstructionDates format=comma10. noseps missing;
+  where not missing(RHS538_endyr);
+  class RHS538_endyr;
+  var RHS538_all_assistedunits;
+  table 
+    /** Rows **/
+    all='Total' RHS538_endyr=' ',
+    /** Columns **/
+    n='Projects'    
+    sum='Assisted Units' * RHS538_all_assistedunits=' ' 
+    ;
+  format RHS538_endyr yearrng.;
+run;
+
+ods csvall close;
+
+/*HOME*/
+ods csvall body="&_dcdata_default_path\RegHsg\Prog\Subsidized_unit_counts_HOME.csv";
+
+title3 "HOME projects and assisted units with subsidies expiring 2015 - 2035";
+footnote1;
+
+proc tabulate data=Work.ConstructionDates format=comma10. noseps missing;
+  where not missing(HOME_endyr);
+  class HOME_endyr;
+  var HOME_all_assistedunits;
+  table 
+    /** Rows **/
+    all='Total' HOME_endyr=' ',
+    /** Columns **/
+    n='Projects'    
+    sum='Assisted Units' * HOME_all_assistedunits=' ' 
+    ;
+  format HOME_endyr yearrng.;
+run;
+
+ods csvall close;
+
+/*Public Housing*/
+ods csvall  body="&_dcdata_default_path\RegHsg\Prog\PH_unit_counts.csv";
+
+title3 "Public housing projects and assisted united with latest construction dates";
+
+proc tabulate data=Work.ConstructionDates format=comma10. noseps missing;
+  where not missing(PHConstructionDate);
+  class ProgCat / preloadfmt order=data;
+  class timecount;
+  var mid_assistedunits moe_assistedunits;
+  table
+    /** Rows **/
+    all='Total' ProgCat=' ',
```

Comment:
Since there is only one type of subsidy we are looking at, you can delete the `all='Total'` part of this line.